docs/content/dependencies/cassandra-deep-storage.md

```diff
@@ -37,8 +37,3 @@ druid.storage.type=c*
 druid.storage.host=localhost:9160
 druid.storage.keyspace=druid
 ```
-
-Use the `druid-development@googlegroups.com` mailing list if you have questions,
```

Comment:
Hm, maybe keep this reference to the original author's email?

Reply:
My rationale here is that it's not typical to include contributors' email addresses in the docs for individual extensions (even if they volunteered!). It's probably better for discussion to happen over the list anyway.